### ISSUE TITLE
some question

### ISSUE THREAD

**Issue by xdg2016** [jhayes14/GAN#4]
hello,Is this code associated with the paper——Generating steganographic images via adversarial
training? Why can't I find the Bob network? thank you.^_^

**Comment by jhayes14** [comment]
No, sorry I originally meant to release the code at the same time as the paper, however this last few months have been very busy. I will hopefully find time in December and have it out before the end of the year.
